### ISSUE TITLE
Broken docstring

### ISSUE THREAD

**Issue by ajshajib** [lenstronomy/lenstronomy#457]
https://github.com/lenstronomy/lenstronomy/blob/5523adf4b3e6e376ae12cdfa4bd05b2dbe7eaba2/lenstronomy/LensModel/Profiles/chameleon.py#L536

The [documentation page](https://lenstronomy.readthedocs.io/en/latest/lenstronomy.LensModel.Profiles.html#lenstronomy.LensModel.Profiles.chameleon.DoubleChameleonPointMass.function) is broken for this function, possibly due to the `#` symbol in the docstring.

**Comment by sibirrer** [comment]
Good catch! The reason is that sphinx demands a blank line between the description text and the :param <variable>: part. I'll update and include it in my next PR
